autopilot/financial-services/bankruptcy.ipynb

```diff
@@ -0,0 +1,704 @@
+{
```

Comment:
<p>prefer f-strings here and elsewhere:</p><p> </p><pre class="ql-syntax" spellcheck="false">test_data_s3_path = session.upload_data(path=test_file, key_prefix=f"{prefix}/test")
print(f"Test data uploaded to: {test_data_s3_path}")
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1827/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>